pipelined/bevy_pbr2/src/render/light.rs

```diff
@@ -273,14 +288,190 @@ impl SpecializedPipeline for ShadowPipeline {
     }
 }
 
-// TODO: ultimately these could be filtered down to lights relevant to actual views
+pub fn update_directional_light_frusta(
```

Comment:
These don't touch render types. I think they should live in the top level light module.

Reply:
Done.